CHANGELOG.md

```diff
@@ -57,6 +57,26 @@ Deprecated names
 New modules
 -----------
 
+* Bundles for left- and right- actions:
+  ```
+  Algebra.Action.Bundles
+  ```
+
+* The free `List` actions over a `SetoidAction`:
+  ```
+  Algebra.Action.Construct.Free
+  ```
+
+* The left- and right- regular actions (of a `Monoid`) over itself:
+  ```
+  Algebra.Action.Construct.Self
```

Comment:
Well, @MatthewDaggitt suggested `Self`, and while I might have thought `Regular` might have been better, here I ended up thinking that a 'neutral' name, *not* informed by existing mathematical practice/conventional usage , might be preferable. It's clear that throughout this PR my instinct were pulling me in multiple directions... not always for the better!

Reply:
I guess I didn't put 2 and 2 together because `Construct` appears in between `Action` and `Self` and I didn't get "self action"! I can get behind this name, now that it has been explained.